packages/StatusTracker/src/components/Point/Point.js

```diff
@@ -60,6 +60,7 @@ const Point = React.forwardRef((props, ref) => {
           onFocus={handleTooltipOpen}
           onBlur={handleTooltipClose}
           kind={kind}
+          isDropdown
```

Comment:
`isDropdown === false` by default? 

If so, then this would be considered a braking-change (major version bump). Just a heads-up.

Reply:
This is an internal prop for css, apis are still same, only if you didn't put an Overflow menu inside a Point, we will show this white button-like trigger